### PR TITLE
Triage/Excluding tests

### PR DESCRIPTION
**Pull request by adam-thorpe**
- Added issue for VMSupportsCS8 and un-excluded on 13
- Added issue for TimSortStackSize2
- Added issue for SpliteratorCollisions
- Excluded CondyDescTest on 13
- Excluded httpclient tests on 11 that were only excluded on 13
- Excluded SplittableRandomTest on 11 which was only excluded on 13
- Excluded ClassForNameLeak on 8
- Added issue for TimedAcquireLeak
- Added issue for TestLoggerWeakRefLeak
- Unexcluded CustomSecurityManager

Signed-off-by: Adam Thorpe <adam.thorpe@ibm.com>

### CODE REVIEW

File: openjdk/ProblemList_openjdk11-openj9.txt [hunk 1]
```diff
@@ -281,18 +284,18 @@ java/rmi/transport/runtimeThreadInheritanceLeak/RuntimeThreadInheritanceLeak.jav
...
-java/util/concurrent/atomic/VMSupportsCS8.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/Spliterator/SpliteratorCollisions.java	https://github.com/eclipse/openj9/issues/7090	generic-all
+java/util/concurrent/atomic/VMSupportsCS8.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1347	generic-all
 java/util/concurrent/forkjoin/FJExceptionTableLeak.java	https://github.com/eclipse/openj9/issues/3209	generic-all
-java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/eclipse/openj9/issues/7125	macosx-all,linux-all
 java/util/concurrent/ArrayBlockingQueue/WhiteBox.java      https://github.com/eclipse/openj9/issues/5988    macosx-all
 java/util/logging/LogManager/TestLoggerNames.java https://github.com/eclipse/openj9/issues/4561 generic-all
-java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1352	generic-all
```
Comment (smlambert) [inline openjdk/ProblemList_openjdk11-openj9.txt]: You can add this 1352 link to the ones marked in https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297, close 1352 and use the 1297 link for tests that are permanent excludes due to them being hotspot specific.

File: openjdk/ProblemList_openjdk13-openj9.txt [hunk 2]
```diff
@@ -302,15 +302,14 @@ java/rmi/transport/runtimeThreadInheritanceLeak/RuntimeThreadInheritanceLeak.jav
...
 java/util/BitSet/stream/BitSetStreamTest.java https://github.com/eclipse/openj9/issues/4720 linux-all
-java/util/Spliterator/SpliteratorCollisions.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/Spliterator/SpliteratorCollisions.java	https://github.com/eclipse/openj9/issues/7090	generic-all
 java/util/concurrent/ArrayBlockingQueue/WhiteBox.java 	https://github.com/eclipse/openj9/issues/5988 	macosx-all
-java/util/concurrent/atomic/VMSupportsCS8.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
 java/util/concurrent/forkjoin/FJExceptionTableLeak.java	https://github.com/eclipse/openj9/issues/3209	generic-all
-java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/util/concurrent/locks/Lock/TimedAcquireLeak.java	https://github.com/eclipse/openj9/issues/7125	macosx-all,linux-all
+java/util/logging/TestLoggerWeakRefLeak.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1352	generic-all
```
Comment (smlambert) [inline openjdk/ProblemList_openjdk13-openj9.txt]: same comment as above, let's amalgamate all non-applicable permanent excludes under 1297

File: openjdk/ProblemList_openjdk8-openj9.txt [hunk 3]
```diff
@@ -303,13 +303,13 @@ java/util/Spliterator/SpliteratorCollisions.java		https://github.com/eclipse/ope
...
 java/util/Timer/NameConstructors.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/821	windows-all
-java/util/concurrent/atomic/VMSupportsCS8.java		https://github.com/eclipse/openj9/issues/1131	generic-all
+java/util/concurrent/atomic/VMSupportsCS8.java		https://github.com/AdoptOpenJDK/openjdk-tests/issues/1347	generic-all
 java/util/concurrent/locks/Lock/TimedAcquireLeak.java		https://github.com/eclipse/openj9/issues/1131	generic-all
 java/util/concurrent/ThreadLocalRandom/ThreadLocalRandomTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/830	linux-all
 java/util/concurrent/ThreadPoolExecutor/CoreThreadTimeOut.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/821	windows-all
 java/util/logging/CheckZombieLockTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/830	generic-all
 java/util/logging/FileHandlerMaxLocksTest.java		https://github.com/eclipse/openj9/issues/1131	generic-all
-java/util/logging/TestLoggerWeakRefLeak.java		https://github.com/eclipse/openj9/issues/1131	generic-all
+java/util/logging/TestLoggerWeakRefLeak.java		https://github.com/AdoptOpenJDK/openjdk-tests/issues/1352	generic-all
```
Comment (smlambert) [inline openjdk/ProblemList_openjdk8-openj9.txt]: #1297

File: openjdk/ProblemList_openjdk11-openj9.txt (hunk 1 above)
Comment (adam-thorpe) [reply openjdk/ProblemList_openjdk11-openj9.txt]: Yes, I was waiting for clarification if these tests were hotspot specific or not before adding them to the permanent exclude list. So are these tests both to be perma excluded?

Comment (adam-thorpe) [reply openjdk/ProblemList_openjdk11-openj9.txt]: When i say these i mean this test on all 3 lists and also https://github.com/AdoptOpenJDK/openjdk-tests/issues/1347

**Comment by smlambert** [comment]
I will review one more time and merge once you resolve the conflicts please.  :)

**Comment by adam-thorpe** [comment]
Done :)

**Pull request merged**